src/Providers/ConsoleServiceProvider.php

```diff
@@ -24,6 +27,20 @@ public function boot()
      */
     public function register()
     {
+        $this->app->singleton('CachetHQ\Cachet\Console\Commands\FixPermissionsCommand', function ($app) {
+            $storageDirectory = $app->path['storage'];
+            $databaseDirectory = $app->path['database'];
+            $databasePath = Config::get('database.connections.sqlite.database');
+            $databaseDefault = Config::get('database.default');
+
+            return new FixPermissionsCommand(
```

Comment:
That's a soft limit. If we don't use one line, there's no point in doing the variable asignments above.